pages/slack.md

```diff
@@ -0,0 +1,18 @@
+title: Slack
+---
+
```

Comment:
Could we have an introductory section here?  How about:

"This year we're experimenting with a conference [Slack](https://slack.com/), for conference delegates to communicate with each other and with the organisers during the conference.

All delegates should receive an email invitation to join the team.  [Get in touch](/contact/) if you think you should have received one but haven't.

We plan to close the team on November 19th, two months after the end of the conference."